vhdbuilder/release-notes/AKSCBLMarinerV2/gen2fips/latest.txt

```diff
@@ -1,6 +1,6 @@
-Starting build on  Wed Dec 4 18:41:51 UTC 2024
+Starting build on  Sun Dec 15 09:06:41 UTC 2024
 Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
-  - oras version 1.2.0
+  - oras version 1.2.1
```

Comment:
nice hopefully this fixes that CVE we've had forever

Reply:
Indeed lol.